doc/dev/installation.rst

```diff
@@ -70,6 +70,79 @@ Building from source
 To build Catalyst from source, developers should follow the instructions provided below for building
 all three modules: the Python frontend, the MLIR compiler, and the runtime library.
 
+
+Just Tell Me How To Build Catalyst From Source
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(This section title is a tribute to the llvm new pass manager guide's section `Just Tell Me How To Run The Default Optimization Pipeline With The New Pass Manager <https://llvm.org/docs/NewPassManager.html#just-tell-me-how-to-run-the-default-optimization-pipeline-with-the-new-pass-manager>`_)
+
+On **Linux Debian/Ubuntu**:
```

Comment:
@josh146 What do you think of this new section?

Reply:
I just had a look, it's a good idea! I have a couple of suggestions, to make this a bit easier to read/parse.

1. Make this section top-level, rather than nested. E.g, have the table of contents go

   - Dev Containers
   - Building from source _(this is your new section)_
   - Detailed source compilation guide
  
2. Since `#` does not represent a comment in (most) console shells, replacing this symbol with run that _does_ represent comments, so that the entire block can be copied and pasted.

3. Using Sphinx-tabs to allow a user to see only the commands they are interested in.

Reply:
Isn't `#` the comment character in bash? e.g. https://jax.readthedocs.io/en/latest/installation.html

Reply:
@paul0403 oops, that's my mistake, not sure why I forgot that 🤦 Ignore point number 2!